dspace-server-webapp/src/test/java/org/dspace/app/rest/AuthorizationRestRepositoryIT.java

```diff
@@ -1517,6 +1525,873 @@ public void findByObjectAndFeatureInternalServerErrorTest() throws Exception {
             .andExpect(status().isInternalServerError());
     }
 
+    @Test
+    /**
+     * Verify that the search by multiple objects and features works properly in allowed scenarios:
+     * - for an administrator
+     * - for an administrator that want to inspect permission of the anonymous users or another user
+     * - for a logged-in "normal" user
+     * - for anonymous
+     *
+     * @throws Exception
+     */
+    public void findByMultipleObjectsAndFeaturesTest() throws Exception {
```

Comment:
In this test, you regularly add/remove features that you're checking. I'd opt to always include all the features that you use throughout the test. This would make it so that the first REST call asks for the same features as the last REST call and that gives us a much cleaner overview of everything that the code handles.